gobblin-core/src/main/java/gobblin/source/extractor/utils/InputStreamCSVReader.java

```diff
@@ -49,7 +49,7 @@ public InputStreamCSVReader(BufferedReader input) {
 
   public InputStreamCSVReader(String input) {
     this(new InputStreamReader(new ByteArrayInputStream(input.getBytes()),
-        Charset.forName(ConfigurationKeys.DEFAULT_CHARSET_ENCODING)));
+        Charset.forName(ConfigurationKeys.DEFAULT_CHARSET_ENCODING)), ',', '\"');
```

Comment:
Not sure why this is needed since the original code does exactly the same thing by calling this(input, ',', '\"');
@narasimhareddyv Could you please take a look? Thanks!